src/scenario/data.h

```diff
@@ -81,10 +81,35 @@ enum {
     ALLOWED_BUILDING_MONUMENTS = 48,
 };
 
-struct win_criteria_t {
+#define MAX_WIN_CRITERIA 10
+
+typedef enum {
+    WIN_CRITERIA_NONE,
+    WIN_CRITERIA_POPULATION_MINIMUM,
+    WIN_CRITERIA_CULTURE,
+    WIN_CRITERIA_PROSPERITY,
+    WIN_CRITERIA_PEACE,
+    WIN_CRITERIA_FAVOR,
+    WIN_CRITERIA_SURVIVAL_YEARS,
+    WIN_CRITERIA_TIME_LIMIT,
+
+    // TODO: Try to implement these
+    WIN_CRITERIA_BUILD_X_BUILDINGS, // Build at least a quantity of a specific building type
+    WIN_CRITERIA_EVOLVE_X_HOUSES,   // Build at least X houses of the given level or higher
+    WIN_CRITERIA_EARN_X_DENARII,    // Have at least X denarii in city coffers
```

Comment:
WIN_CRITERIA_EARN_X_DENARII should perhaps be renamed WIN_CRITERIA_HAVE_X_DENARII ??

WIN_CRITERIA_EARN_X_DENARII sounds to me more like it is referring to income per year.
Perhaps WIN_CRITERIA_EARN_X_DENARII_ONE_YEAR can also be such a future goal?

Either way, minor thing.

Reply:
Good suggestion, and I see your reasoning.